requirements.txt

```
@@ -4,7 +4,7 @@ numpy>=1.16.6
 torch>=1.3
 future>=0.17.1  # required for builtins in setup.py
 # pyyaml>=3.13
-PyYAML>=5.1  # OmegaConf requirement >=5.1
+PyYAML>=5.1, !=5.4  # OmegaConf requirement >=5.1
```

```suggestion
PyYAML>=5.1,!=5.4  # OmegaConf requirement >=5.1
```

Not sure if this space is problematic